src/openforms/formio/tests/test_api_fileupload.py

```diff
@@ -106,21 +106,6 @@ def test_upload_empty(self):
         )
         self.assertEqual(response.content_type, "text/plain")
 
-    def test_content_inconsistent_with_mime_type(self):
-        self._add_submission_to_session(self.submission)
-
-        url = reverse("api:formio:temporary-file-upload")
-        file = SimpleUploadedFile("pixel.png", b"GIF89a", content_type="image/png")
-        response = self.client.post(url, {"file": file}, format="multipart")
-
-        self.assertContains(
-            response,
-            _("The provided file is not a {file_type}.").format(
-                **{"file_type": ".png"}
-            ),
-            status_code=status.HTTP_400_BAD_REQUEST,
-        )
-
```

Comment:
why is this test removed?

How I read this testing code right now is that it seems possible to make a temporary upload with a file whose content doesn't match the declared content type. That should not never be allowed, we need to stop the bogus input as soon as possible.